### PR TITLE
Add _TZ3000_qeuvnohg Tongou TO-Q-SY1-JZT

### PR DESCRIPTION
**Pull request by arsenyspb**
Original [issue/2005](https://github.com/zigpy/zha-device-handlers/issues/2005) and [pull/2008](https://github.com/zigpy/zha-device-handlers/pull/2008). I'm not able to push to @zleba 's [zleba:tuyaCircuitBreaker](https://github.com/zleba/zha-device-handlers/tree/tuyaCircuitBreaker) as not a maintaner, therefore this PR.

Following @javicalle 's hint [here](https://github.com/zigpy/zha-device-handlers/issues/1764#issuecomment-1272390644), and once quirk is installed this problems are gone [(tested)](https://www.evernote.com/shard/s151/sh/9f06b277-d45f-4ff7-b4d3-d8d1287208b5/IXMgWdpsTeE8GjaIfgHdQwklEFCfUQhEXuyYIDv3WLzsrfltSovN_uYm0Q) through `DIVISOR` and `MULTIPLIER` in `tuya/__init__.py`.

### CODE REVIEW

File: zhaquirks/tuya/ts011f_plug.py [hunk 1]
```diff
@@ -1141,3 +1142,69 @@ class Plug_2AC_var03(CustomDevice):
             },
         },
     }
+class Plug_CB_Metering(EnchantedDevice):
```
Comment (javicalle) [inline zhaquirks/tuya/ts011f_plug.py:1147]: Add 2 blank lines before the class declaration. Like this:
```python
                OUTPUT_CLUSTERS: [],
            },
        },
    }


class Plug_CB_Metering(EnchantedDevice):
    """Circuit breaker with monitoring, e.g. Tongou TO-Q-SY1-JZT. First one using this definition was _TZ3000_qeuvnohg."""

    signature = {
        MODEL: "TS011F",
```

File: zhaquirks/tuya/ts011f_plug.py [hunk 2]
```diff
@@ -1141,3 +1142,69 @@ class Plug_2AC_var03(CustomDevice):
...
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_PLUG_IN_UNIT,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Identify.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                    TemperatureMeasurement.cluster_id, # Note: no temperature sensor claimed in manual and it's constant -6.3C
```
Comment (javicalle) [inline zhaquirks/tuya/ts011f_plug.py]: As states, 2 white spaces before the comment. Like this:
```python
                INPUT_CLUSTERS: [
                    Basic.cluster_id,
                    Identify.cluster_id,
                    Groups.cluster_id,
                    Scenes.cluster_id,
                    OnOff.cluster_id,
                    TemperatureMeasurement.cluster_id,  # Note: no temperature sensor claimed in manual and it's constant -6.3C
                    Metering.cluster_id,
                    ElectricalMeasurement.cluster_id,
                    TuyaZBE000Cluster.cluster_id,
                    TuyaZBExternalSwitchTypeCluster.cluster_id,
                ],
```

**Review by javicalle — CHANGES_REQUESTED** [review]
Checks fixes

**Review by javicalle — APPROVED** [review]
LGTM

**Comment by arsenyspb** [comment]
@javicalle 🙏  kudos, you folks maintain the project really well, - I was able to learn everything I needed in a matter of hours (...or maybe it's just a productive Sunday).

How often you merge (...looks like to dev), what to RTFM about your release?

Basically, - when to remove my local quirk from `/config/custom_zha_quirks`

**Comment by javicalle** [comment]
Sorry I'm just a collaborator. I'm not in charge to merge or release versions.
Usually before a new release most of the small PRs are merged, but there is no ETA for the versions.

**Comment by arsenyspb** [comment]
Cool will keep an eye on this one and will remove based on release notes from upstream.

**Comment by zleba** [comment]
Hi @arsenyspb, actually the measurement of the power (W), power consumption (kWh) and current (A) works well for me after using the quirk in the PR.
But I haven't managed to get the "Power factor" working (cos th).
If this entry is not required, you can go ahead.
It can be calculated as:
power / (current * voltage)
To my knowledge, there is no chance to know if the connected device is more like a capacitor or inductor...

File: zhaquirks/tuya/ts011f_plug.py [hunk 3]
```diff
@@ -1141,3 +1142,71 @@ class Plug_2AC_var03(CustomDevice):
             },
         },
     }
+
+
+class Plug_CB_Metering(EnchantedDevice):
```
Comment (TheJulianJES) [inline zhaquirks/tuya/ts011f_plug.py:1147]: just a note: Class names are normally `CamelCase` in Python and don't include underscores, but I see that underscores are used everywhere here, so let's keep it like this (until changed completely?)

**Review by TheJulianJES — APPROVED** [review]
LGTM. Thanks @arsenyspb and @zleba

**Comment by TheJulianJES** [comment]
Changes will likely land in Home Assistant 2023.4.0 (hopefully even during the beta period).
Normally, library bumps only make it into "hotfix" releases if somewhat critical issues are fixed.

(See https://developers.home-assistant.io/ "Upcoming Events" for the HA Core release schedule)

> But I haven't managed to get the "Power factor" working (cos th).
If this entry is not required, you can go ahead.
To my knowledge, there is no chance to know if the connected device is more like a capacitor or inductor...

Yeah, ZHA (HA Core) shouldn't create the power factor entity if it's not supported. Just disable it for now.

**Pull request merged**